### DISCUSSION TITLE
Temperature/Lux

### DISCUSSION THREAD

**Discussion by GeorgeCastanza** [andesse/hue-clip-api.node-red-flows#19] [General]
Is there any method of grabbing lux/temperature using the API v2?

**Comment by andesse** [comment]
Hi, 
yes it is, I recommend as also suggested in the readme, create a developer account at hue for the api documentation.
If you request for the whole bridge resource, it should also show up. 

https://developers.meethue.com/

**Comment by andesse** [comment]
@GeorgeCastanza as I am Reading trough the discussion again, I just realized that the normal motion sensor updates will show both. Just switch them out for your needs. 

Greetings 
Andreas
